<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleParaSet</class>
 <widget class="QWidget" name="VehicleParaSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="VehicleParaSet_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>440</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_Sandforbidden">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>撒沙禁止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_EBStop">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>紧急广播停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_EB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>紧急广播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_FBRMC1">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>强迫制动缓解</string>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_Label_HMiCT_TempStart">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_TempStartset">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_TempStartm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_TempStartp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_TempStopp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_TempStopm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_Label_HMiCT_TempStop">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  12px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>开启温度(-10℃~5℃):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  12px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>关闭温度(0℃~15℃):
且不得低于开启温度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_MonthStop1p">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>伴热关闭起始月份:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_label_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  12px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>伴热关闭终止月份:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_Label_HMiCT_MonthStop2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_MonthStop2m">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_MonthStopSet_B1">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_MonthStop2p">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleParaSet_Button_HMiCT_MonthStop1m">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_Label_HMiCT_MonthStop1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_background_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
border:2px solid;
border-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75  12px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>    蓄电池伴热温度设定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleWheelset_Button_NextPage">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleWheelset_realaccmilelbl_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>2/2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleWheelset_Button_PrePage">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VehicleWheelset_Button_return">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>400</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75  18px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>当日牵引能耗：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTCUComsuption">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 45, 45);
font: 75  18px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>VehicleParaSet_background</zorder>
  <zorder>VehicleParaSet_background_2</zorder>
  <zorder>VehicleParaSet_Button_Sandforbidden</zorder>
  <zorder>VehicleParaSet_Button_EBStop</zorder>
  <zorder>VehicleParaSet_Button_EB</zorder>
  <zorder>VehicleParaSet_Button_FBRMC1</zorder>
  <zorder>VehicleParaSet_Label_HMiCT_TempStart</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_TempStartset</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_TempStartm</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_TempStartp</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_TempStopp</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_TempStopm</zorder>
  <zorder>VehicleParaSet_Label_HMiCT_TempStop</zorder>
  <zorder>VehicleParaSet_label_6</zorder>
  <zorder>VehicleParaSet_label_7</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_MonthStop1p</zorder>
  <zorder>VehicleParaSet_label_8</zorder>
  <zorder>VehicleParaSet_label_9</zorder>
  <zorder>VehicleParaSet_Label_HMiCT_MonthStop2</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_MonthStop2m</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_MonthStopSet_B1</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_MonthStop2p</zorder>
  <zorder>VehicleParaSet_Button_HMiCT_MonthStop1m</zorder>
  <zorder>VehicleParaSet_Label_HMiCT_MonthStop1</zorder>
  <zorder>VehicleParaSet_label_10</zorder>
  <zorder>VehicleWheelset_Button_NextPage</zorder>
  <zorder>VehicleWheelset_realaccmilelbl_2</zorder>
  <zorder>VehicleWheelset_Button_PrePage</zorder>
  <zorder>VehicleWheelset_Button_return</zorder>
  <zorder>label</zorder>
  <zorder>LBLTCUComsuption</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
